### PR TITLE
Add RAKwireless devices

### PR DESCRIPTION
**Pull request by jmarcelino**
#### Summary

...

#### Changes

Add RAKwireless and 2 node devices
Wisblock RAK4631
RAK7431 Modbus to LoRaWAN bridge

#### Notes for Reviewers

...

#### Release Notes

- ...

### CODE REVIEW

File: vendor/rakwireless/rak4631-868-profile.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+vendorProfileID: 4631
+supportsClassB: false
+supportsClassC: false
+macVersion: 1.0.2
+regionalParametersVersion: RP001-1.0.2
+supportsJoin: true
+maxEIRP: 22
```
Comment (johanstokking) [inline vendor/rakwireless/rak4631-868-profile.yaml:7]: This is too high for EU868 region

File: vendor/rakwireless/rak7431-868-profile.yaml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+# Ping slot data rate index
+pingSlotDataRateIndex: 0
+# Ping slot frequency (MHz). Set to 0 if the band supports ping slot frequency hopping.
+pingSlotFrequency: 869.525
+supportsClassC: true
+classCTimeout: 60
+macVersion: 1.0.3
+regionalParametersVersion: RP001-1.0.3-RevA
+supportsJoin: true
+maxEIRP: 20
```
Comment (johanstokking) [inline vendor/rakwireless/rak7431-868-profile.yaml:16]: Idem

**Review by johanstokking — COMMENTED** [review]
Thanks, just one remark/question!

File: vendor/rakwireless/rak4631-868-profile.yaml (hunk 1 above)
Comment (jmarcelino) [reply vendor/rakwireless/rak4631-868-profile.yaml]: Thanks Johan, I wasn't sure if that was what the hardware was capable of or as limited by the firmware for the region.
Fixed now.

File: vendor/rakwireless/rak7431-868-profile.yaml [hunk 3]
```diff
@@ -0,0 +1,17 @@
+vendorProfileID: 7431
```
Comment (benolayinka) [inline vendor/rakwireless/rak7431-868-profile.yaml:1]: Does RAK have different LoRaWAN Alliance Vendor Profile IDs? In the index, it looks like your VendorID is 570: https://github.com/TheThingsNetwork/lorawan-devices/blob/master/vendor/index.yaml#L904

Comment (jmarcelino) [reply vendor/rakwireless/rak7431-868-profile.yaml:1]: Thanks, your description suggests this was our own number
"# Vendor profile ID, can be freely issued by the vendor"
So is this actually our LoRaWAN alliance ID? Not device specific?

Comment (johanstokking) [reply vendor/rakwireless/rak7431-868-profile.yaml:1]: This is device profile specific, so it looks good to me.

**Review by johanstokking — APPROVED** [review]

**Pull request merged**